test/rebar3_hex_file_SUITE.erl

```diff
@@ -15,9 +15,23 @@ expand_paths_test(_Config) ->
     File = string:join([Dir2, "bar"], "/"),
     ok = file:write_file(File, ""),
     ?assertEqual([{"bar", File}], rebar3_hex_file:expand_paths(["bar"], "foo")),
-    Exp = [{"foo/bar", File}],
+    Exp = [{"foo", Dir2}, {"foo/bar", File}],
     ?assertEqual(Exp, rebar3_hex_file:expand_paths(["foo*"], Dir)).
 
+expand_once_test(_Config) ->
```

Comment:
There's one ugly truth that effects both mix hex and rebar3 hex, that is if somone gives a double star glob (i.e., `ouch/**`). We could guard against it, perhaps but we would invalidate other "safe" cases.